docs/docs/reference/state-representation.md

```diff
@@ -0,0 +1,1757 @@
+
+- `CartesianPose`
+- `CartesianTwist`
+- `CartesianAcceleration`
+- `CartesianWrench`
+
+### Cartesian pose
+
+The `CartesianPose` class defines only the position and orientation of a frame.
+
+It provides the following constructors:
+
+<Tabs groupId="programming-language">
+<TabItem value="c++" label="C++">
+
+```cpp
+CartesianPose::Identity("name", "reference_frame");
+CartesianPose::Random("name", "reference_frame");
+
+// the position can be supplied to the constructor as a vector or as individual terms (with null orientation)
+xyz = Eigen::Vector3d(1.0, 2.0, 3.0);
+CartesianPose("name", xyz, "reference_frame");
+CartesianPose("name", 1.0, 2.0, 3.0, "reference_frame");
+
+// the orientation can be supplied to the constructor as a vector or as individual terms (with zero displacement)
+q = Eigen::Quaterniond(0, 1, 0, 0);
+CartesianPose("name", q, "reference_frame");
+
+// the position and orientation can also be supplied together
+CartesianPose("name", xyz, q, "reference_frame");
+```
+
+</TabItem>
+</Tabs>
+
+Addition and subtraction is supported between `CartesianPose` and `CartesianState`. Recall that these operations are
+not commutative in orientation; the order of operations matters.
+
+The return type of each compatible operation is shown below.
+
+<Tabs groupId="programming-language">
+<TabItem value="c++" label="C++">
+
+```cpp
+CartesianPose r1 = pose + other_pose;
+CartesianState r2 = pose + state;
+CartesianState r3 = state + pose;
+
+CartesianPose r4 = pose - other_pose;
+CartesianState r5 = pose - state;
+CartesianState r6 = state - pose;
+
+pose += other_pose; // equivalent to pose = pose + other_pose
+pose += state; // equivalent to pose = pose + state
+
+pose -= other_pose;
+pose -= state;
+```
+
+</TabItem>
+</Tabs>
+
+A `CartesianPose` can be used to transform any other Cartesian class by applying the position and orientation offset.
+Recall that the frame and reference frames must be compatible according to the chain rule as described in the
+[Cartesian transformation](#cartesian-transforms-changing-the-reference-frame) section.
+
+<Tabs groupId="programming-language">
+<TabItem value="c++" label="C++">
+
+```cpp
+CartesianPose transformed_pose = pose * other_pose;
+CartesianTwist transformed_twist = pose * twist;
+CartesianAcceleration transformed_acceleration = pose * acceleration;
+CartesianWrench transformed_wrench = pose * wrench;
+CartesianState transformed_state = pose * state
+    
+pose *= other_pose; // equivalent to pose = pose * other_pose
+pose *= state; // equivalent to pose = pose * state
+```
+
+</TabItem>
+</Tabs>
+
+The time derivative of a `CartesianPose` is a `CartesianTwist`. Because `CartesianPose` represents a displacement in
+position and orientation, it can be converted into linear and angular velocity through division by a time period.
+
+Operations with time use `std::chrono::duration` types, such as `std::chrono::milliseconds`, `std::chrono::seconds`, or
```

Comment:
Here we should now mention the overloaded `.integrate()` and `.differentiate()` that are much easier to use